doc/vim-iced.txt

```diff
@@ -217,6 +218,48 @@ SHADOW-CLJS~
 <
   To start CLJS REPL, see |vim-iced-shadow-cljs|.
 
+                                              *vim-iced-manual-rebel-readline*
+REBEL READLINE~
+
+  If you use Rebel Readline or want to embed an nREPL in your application,
+  then one approach is to create a `user.clj` with the following:
+>  
+  (ns user (:require [nrepl.server :as nrepl-server]
+                  [clojure.java.io :as io]))
+  
+  (def nrepl-port 7888)
+  (defonce nrepl-server (atom nil))
+
+  (defn cider-middleware
+    "Get cider middleware, see 
+    https://github.com/clojure-emacs/cider-nrepl/issues/447"
+    []
+    (require 'cider.nrepl)
+    (map resolve (var-get (ns-resolve 'cider.nrepl 'cider-middleware))))
+
+  (defn dev-middleware []
+    (mapcat (fn [[ns syms]] (require ns) (map (partial ns-resolve ns) syms))
+    [['refactor-nrepl.middleware ['wrap-refactor]] ['iced.nrepl ['wrap-iced]]]))
+   
+  (defn start-nrepl-server! []
+    (reset!
+      nrepl-server
+        (nrepl-server/start-server :port nrepl-port :handler (nrepl-handler))
```

Comment:
Are you missing the definition of `nrepl-handler`?

Reply:
@liquidz you were right, I missed it out somehow. Updated the PR.